release notes/v0.41.0.md

```diff
@@ -0,0 +1,64 @@
+k6 v0.41.0 is here! :tada: It has relatively few user-facing changes, but includes massive internal improvements that will pave the way [for some awesome features](#roadmap-and-future-plans) in the near future. Unfortunately, we also had to make a few [minor breaking changes](#breaking-changes) to allow us to do that.
+
+## Breaking changes
+
+### Changes in the `url`, `iter` and `vu` system metric tags
+
+As we warned in both the k6 [v0.39.0](https://github.com/grafana/k6/blob/master/release%20notes/v0.39.0.md#refactoring-metrics) and [v0.40.0](https://github.com/grafana/k6/blob/master/release%20notes/v0.40.0.md#refactoring-metrics) release notes, we have been fundamentally [refactoring the metrics sub-systems](#metric-time-series) of k6. We now have efficient support for time series, and that required a few minor user-facing breaking changes:
+
+1. If [URL grouping](https://k6.io/docs/using-k6/http-requests/#url-grouping) is used for HTTP requests (that is, if the `name` tag is somehow specified), then the `url` tag in the resulting `http_req_*` metric samples will also have the same value as the `name` tag. Previously this was done only for [the `cloud` output](https://k6.io/docs/results-output/real-time/cloud/), but now it will be done universally ([#2703](https://github.com/grafana/k6/pull/2703)).
```

Comment:
Nit: this also happens when [`http.url`](https://k6.io/docs/javascript-api/k6-http/urlurl) is used. Which in practice sets the `name` but looks slightly nicer (for simple cases as well).

Reply:
Resolved with the [latest force-push](https://github.com/grafana/k6/compare/0b6ec90828afac81746a6d300c878006c57b5ac7..f68483e91bdb7c70579acea6ada26a549c7a6e6e)